examples/config.example.json

```diff
@@ -1,18 +1,32 @@
 {
     "work_dir": "/absolute/path/to/work_dir",
 
-    "google_play_accounts": {
+    "products": {
         "aurora": {
+            "has_nightly_track": false,
             "service_account": "service-account-which-have-access-to-aurora@iam.gserviceaccount.com",
-            "certificate": "/absolute/path/to/certificate-for-aurora-service-account.p12"
+            "certificate": "/absolute/path/to/certificate-for-aurora-service-account.p12",
```

Comment:
Nit: Even though this is just an example, I'd be in favor of explicitly mentioning: 

```json
"expected_package_names": ["org.mozilla.fennec_aurora"],
```

Otherwise the package name is not checked, as far as I understand.

Reply:
Ah, I see! You're right, this check wasn't present. We may address it in a follow up.

We can predict some of the names: 

* aurora => `org.mozilla.fennec_aurora` 
* beta => `org.mozilla.firefox_beta` 
* release => `org.mozilla.firefox` 

Dep is a particular case, it's mainly `org.mozilla.fennec_aurora`, but it can be anything else, so let's not check it.